package.json

```diff
@@ -8,68 +8,69 @@
     "export": "NODE_ENV=production webpack --config webpack/build.babel.js",
     "preview": "NODE_ENV=production webpack --config webpack/preview.babel.js",
     "lint": "run-p -c lint:*",
-    "lint:js": "eslint --ext .js .",
+    "lint:js": "eslint --ext .js src",
     "lint:css": "stylelint '**/*.(s)css'",
     "release": "shipjs prepare"
   },
   "dependencies": {
-    "algoliasearch": "4.1.0",
+    "algoliasearch": "4.12.2",
     "invariant": "2.2.4",
-    "preact": "10.4.0",
-    "qs": "6.9.3",
-    "react-instantsearch-dom": "6.4.0",
-    "react-router-dom": "5.1.2",
+    "preact": "10.6.6",
+    "qs": "6.10.3",
+    "react-instantsearch-dom": "6.22.0",
+    "react-router-dom": "6.2.2",
     "rheostat": "2.2.0",
-    "search-insights": "1.4.0"
+    "search-insights": "2.2.1"
   },
   "devDependencies": {
-    "@babel/core": "7.9.0",
-    "@babel/preset-env": "7.9.5",
-    "@babel/preset-react": "7.9.4",
-    "@babel/register": "7.9.0",
-    "babel-eslint": "10.1.0",
-    "babel-loader": "8.1.0",
+    "@babel/core": "7.17.5",
+    "@babel/eslint-parser": "7.17.0",
+    "@babel/preset-env": "7.16.11",
+    "@babel/preset-react": "7.16.7",
+    "@babel/register": "7.17.0",
+    "babel-loader": "8.2.3",
     "babel-plugin-strip-invariant": "1.0.0",
-    "clean-webpack-plugin": "3.0.0",
-    "copy-webpack-plugin": "5.1.1",
-    "css-loader": "3.5.2",
-    "eslint": "6.8.0",
-    "eslint-config-algolia": "15.0.0",
-    "eslint-config-prettier": "6.10.1",
-    "eslint-plugin-eslint-comments": "3.1.2",
-    "eslint-plugin-import": "2.20.2",
-    "eslint-plugin-jest": "23.8.2",
-    "eslint-plugin-prettier": "3.1.3",
-    "eslint-plugin-react": "7.19.0",
-    "eslint-plugin-react-hooks": "3.0.0",
-    "html-webpack-plugin": "4.3.0",
+    "copy-webpack-plugin": "10.2.4",
+    "css-loader": "6.6.0",
+    "css-minimizer-webpack-plugin": "3.4.1",
+    "eslint": "8.10.0",
+    "eslint-config-algolia": "20.0.0",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-eslint-comments": "3.2.0",
+    "eslint-plugin-import": "2.25.4",
+    "eslint-plugin-jest": "26.1.1",
+    "eslint-plugin-jsdoc": "37.9.6",
+    "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-prettier": "4.0.0",
+    "eslint-plugin-react": "7.29.2",
+    "eslint-plugin-react-hooks": "4.3.0",
+    "html-webpack-plugin": "5.5.0",
     "ignore-styles": "5.0.1",
-    "mini-css-extract-plugin": "0.9.0",
+    "mini-css-extract-plugin": "2.5.3",
     "module-alias": "2.2.2",
-    "node-sass": "4.13.1",
+    "node-sass": "7.0.1",
```

Comment:
`node-sass` is now deprecated (and causing difficulties on M1 Macs). Could we switch with [`dart-sass`](https://github.com/sass/dart-sass)?